atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -68,7 +71,7 @@ public KeyValueService getKeyValueService() {
     }
 
     public synchronized TimestampService getTimestampService() {
-        DebugLogger.logger.info("Fetching timestamp service from thread {}. This should only happen once.",
+        log.info("Fetching timestamp service from thread {}. This should only happen once.",
```

Comment:
I was wondering whether this logging should, in fact, go to "normal" logs rather than the "Debug" logs. I guess they'd be most noticeable in the normal logs, so this is ok.

As an aside - `ServiceDiscoveringAtlasSupplier` and `TransactionManagers` don't sound like they should be a part of `atlasdb-config`. However, changing this would _definitely_ be a breaking change, as all products use `TransactionManagers.create`.